src/parser/mod.rs

```diff
@@ -16476,6 +16484,22 @@ mod tests {
             }};
         }
 
+        macro_rules! mk_expected_col {
+            ($name:expr) => {
+                IndexColumn {
+                    column: OrderByExpr {
+                        expr: Expr::Identifier($name.into()),
+                        options: OrderByOptions {
+                            asc: None,
+                            nulls_first: None,
+                        },
+                        with_fill: None,
+                    },
+                    operator_class: None,
+                }
+            };
+        }
```

Comment:
Would it make more sense to use a function for this instead? since macros aren't the best for readability